test/Recollateralization.test.ts

```diff
@@ -809,12 +816,6 @@ describe(`Recollateralization - P${IMPLEMENTATION}`, () => {
         await stRSR.connect(addr1).stake(stakeAmount)
       })
 
-      afterEach(async () => {
```

Comment:
Why are we removing this? Seems like a check you dont want to drop.

Reply:
It's not something in 3.0.0 or master, fyi, it's something I added in `multiple-nonce-redemption`. I'm making some tweaks to keep it though, I see an easy way to do it now that doesn't involve changing tests to get them to end in a different state.

Reply:
Yeah noticed later that it wasnt something we had before. Cool.